sliceop/f64s/f64s.go

```diff
@@ -62,3 +62,24 @@ func Find(dst []int, src []float64, f func(v float64) bool) []int {
 
 	return dst
 }
+
+// Take creates a slice with all elements of src indiced by the provided indices.
+// Take uses dst as work buffer, storing elements at the start of the slice.
+// Take clears dst if a slice is passed, and allocates a new slice if dst is nil.
+func Take(dst, src []float64, indices []int) []float64 {
+
+	if dst == nil {
+		dst = make([]float64, 0, len(indices))
+	}
+
+	if len(indices) > len(src) {
+		panic("f64s: length mismatch")
+	}
+
+	dst = dst[:0]
```

Comment:
```suggestion
	dst = dst[:len(indices)]
```
(we should also probably add a test on the length of `dst` before that line)